src/Symfony/Component/Notifier/Bridge/Sevenio/composer.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/sevenio-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Seven.io Notifier Bridge",
+    "keywords": ["sms", "sevenio", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Frank Nägler",
+            "email": "frank@naegler.hamburg"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/notifier": "^6.4|^7.0"
```

```suggestion
        "symfony/notifier": "^7.1"
```
This is required to make the tests green

---

I don't think so.

---

Otherwise the UnsupportedException test is not passing, because it is not included in an older version

---

besides that, I guess I have to switch the target branch, right?

---

Yes, rebase on 7.1 and change the target branch to 7.1